rust/cyclotron-node/src/lib.rs

```diff
@@ -422,6 +425,33 @@ fn set_parameters(mut cx: FunctionContext) -> JsResult<JsNull> {
     Ok(cx.null())
 }
 
+fn set_blob(mut cx: FunctionContext) -> JsResult<JsNull> {
+    let arg = cx.argument::<JsString>(0)?.value(&mut cx);
+    let job_id: Uuid = arg
+        .parse()
+        .or_else(|_| cx.throw_error(format!("invalid job id: {}", arg)))?;
+
+    // Tricky - we have to support passing nulls here, because that's how you clear the blob.
+    let blob = cx.argument::<JsValue>(1)?;
+    let blob: Option<Vec<u8>> = if blob.is_a::<JsNull, _>(&mut cx) {
```

Comment:
As an aside, I'm surprised this works. Wouldn't these be `undefined` and not `null` most of the time? (Although we should probably support either one clearing it.) I'm surprised it doesn't fall into the following code and throw?